challenges/dog-photo-gallery/script.js

```diff
@@ -0,0 +1,61 @@
+const gallery = document.createElement("div");
+gallery.classList.add("photo-gallery");
+const displayDogBtn = document.createElement("button");
+displayDogBtn.textContent = "Click to display random dog";
+const ul = document.createElement("ul");
+const loadMsgEl = document.createElement("div");
+loadMsgEl.classList.add("loading-msg");
+
+document.body.append(gallery, displayDogBtn, ul, loadMsgEl);
```

Comment:
The requirements mention adding two buttons. You currently have one button for displaying a random dog image. What functionality can you think of for a second button? (eg. clearing the gallery? fetching more than one image?)

Reply:
Adding a button for clearing the gallery would be a good option which I am going to implement.